vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -56,6 +56,7 @@ void draw_game_install_dialog(GuiState &gui, HostState &host) {
     }
 
     static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+    static bool delete_game_file = false;
```

Comment:
stop using `static` for non-const variables
make this part of `GuiState` or something